maestro-ios-driver/src/main/kotlin/util/IOSDevice.kt

```diff
@@ -19,7 +19,7 @@ data class DeviceCtlResponse(
     @JsonIgnoreProperties(ignoreUnknown = true)
     data class Device(
         val identifier: String,
-        val deviceProperties: DeviceProperties,
+        val deviceProperties: DeviceProperties?,
```

Comment:
Not sure I understand the cause or the fix. It’s making 3 properties nullable. Why weren’t they before? Those aren’t new attributes afaik, when were they introduced?

Reply:
These fields were added along with physical device support and are used to show the connected device's description. In my local testing, all three values were always present, so I overlooked the possibility that any of them could be null. This update now handles those cases properly.

I'm also in touch with a customer to look this.